integration-cli/docker_cli_exec_test.go

```diff
@@ -595,3 +595,13 @@ func (s *DockerSuite) TestExecOnReadonlyContainer(c *check.C) {
 		c.Fatalf("exec into a read-only container failed with exit status %d", status)
 	}
 }
+
+// #15750
+func (s *DockerSuite) TestExecStartFails(c *check.C) {
+	name := "exec-15750"
+	dockerCmd(c, "run", "-d", "-t", "--name", name, "busybox", "/bin/sh")
```

Comment:
I think if you us "top" instead of "sh" then you can just do: run -d busybox top